### DISCUSSION TITLE
pyswmm show error when the file could be simulated in SWMM5

### DISCUSSION THREAD

**Discussion by xuhotting** [pyswmm/pyswmm#426] [Q&A] [answered]
**Describe the bug**
When I use pyswmm to run an file, it shows an error: "ERROR 200: one or more errors in input file."
However, this file could be simulated in SWMM5.
How to solve this error?

**To Reproduce**
import pyswmm
with pyswmm.Simulation(r"D:\swmm_after\after_5.inp", 'model.out') as sim:
    sim.start()

**Screenshots**
![image](https://user-images.githubusercontent.com/115472197/232177911-0d2756f5-cb2f-419d-81cf-f8e26a6f08b7.png)

**Comment by bemcdonnell** [comment]
What version of pyswmm are you using?

**Comment by xuhotting** [comment]
> What version of pyswmm are you using?

It's 1.4.0. and the python version is 3.10.7

**Comment by bemcdonnell** [comment, marked as answer]
Try adding a RPT file in your simulation class and run it. The RPT file will tell you what SWMM’s errors are, specifically

**Reply by xuhotting** [reply]
Thanks for your advice!
I have used the code without adding RPT file in Colab, and it successfully ran yesterday.
